### PR TITLE
Add a file with a bad copyright year

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

**Comment by mattmoor-sockpuppet** [comment]
PR event: opened

File: cmd/doc.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2018 The Knative Authors
```
Comment (mattmoor-sockpuppet) [inline cmd/doc.go]:
```suggestion
Copyright 2019 The Knative Authors
```

**Comment by mattmoor-sockpuppet** [comment]
PR event: reopened

**Comment by mattmoor-sockpuppet** [comment]
PR event: closed

File: cmd/doc.go (hunk 1 above)
Comment (mattmoor-sockpuppet) [inline cmd/doc.go]:
```suggestion
Copyright 2019 The Knative Authors
```

File: foo.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2018 Bad Author
```
Comment (mattmoor-sockpuppet) [inline foo.yaml]:
```suggestion
# Copyright 2019 Bad Author
```

**Comment by mattmoor-sockpuppet** [comment]
PR event: synchronize

**Comment by mattmoor-sockpuppet** [comment]
PR event: labeled

**Comment by mattmoor-sockpuppet** [comment]
PR event: unlabeled

**Comment by mattmoor-sockpuppet** [comment]
PR event: labeled

**Comment by mattmoor-sockpuppet** [comment]
PR event: unlabeled

**Comment by mattmoor-sockpuppet** [comment]
PR event: labeled

File: cmd/doc.go (hunk 1 above)
Comment (mattmoor) [inline cmd/doc.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

Comment (mattmoor) [inline cmd/doc.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

Comment (mattmoor) [inline cmd/doc.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

Comment (mattmoor) [inline cmd/doc.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: cmd/doc.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (mattmoor) [inline cmd/doc.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

File: cmd/doc.go (hunk 1 above)
Comment (mattmoor) [inline cmd/doc.go]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

Comment (mattmoor) [inline cmd/doc.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

**Pull request closed without merging**